nilearn/plotting/tests/test_displays.py

```diff
@@ -271,7 +289,7 @@ def test_user_given_cmap_with_colorbar(img):
 
 @pytest.mark.parametrize("display", [OrthoSlicer, LYRZProjector])
 def test_data_complete_mask(display):
-    """This special case test is due to matplotlib 2.1.0.
+    """Test for a special case due to matplotlib 2.1.0.
```

Comment:
Maybe leave for another PR but maybe this patch and hence test are not needed anymore given that we support mpl>=3.3.0

Reply:
let's flag this for another PR.